src/pkg/packager/actions.go

```diff
@@ -200,6 +200,15 @@ func actionCmdMutation(cmd string, shellPref types.ZarfComponentActionShell) (st
 		return cmd, err
 	}
 
+	// if another binary is calling Zarf, use the system Zarf
+	regex, err := regexp.Compile("^.*zarf(\\.exe)?$")
```

Comment:
```suggestion
	// See https://regex101.com/r/O4Ngia/1
	regex, err := regexp.Compile(`^.*zarf(\.exe)?$`)
```
Just a small change to formatting - we usually provide regex101 links for regex's and use ` instead of " for the regex string.